apps/api/src/app/portfolio/portfolio-calculator.ts

```diff
@@ -199,16 +200,10 @@ export class PortfolioCalculator {
       }) ?? [];
 
     const currencies: { [symbol: string]: string } = {};
-    const dates: Date[] = [];
     const dataGatheringItems: IDataGatheringItem[] = [];
     const firstIndex = transactionPointsBeforeEndDate.length;
 
-    let day = start;
-
-    while (isBefore(day, end)) {
-      dates.push(resetHours(day));
-      day = addDays(day, step);
-    }
+    const dates = eachDayOfInterval({ start, end }, { step });
```

Comment:
For `api/v2/portfolio/performance?range=1d` I just get a horizontal line (0). It seems to work as before if I do the following:

```
const dates = eachDayOfInterval({ start, end }, { step }).map((date) => {
  return resetHours(date);
});
```